.buildkite/scripts/aws-lambda-packaging.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+
+function __package_lambda() {
+  local package_path="$1"
```

Comment:
I'd be tempted to make the packaging and deployment of lambdas a task that you run via yarn like: https://github.com/wellcomecollection/wellcomecollection.org/blob/master/.buildkite/pipeline.yml#L81